recipes/tower-cli/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.8.0" %}
+{% set sha256 = "c0d08bbf3059296e4819d33c6ff9282cfdd4d97087213c2d9aa0b5943ced60f4" %}
+
+package:
+  name: tower-cli
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  - url: https://github.com/seqeralabs/tower-cli/releases/download/v{{ version }}/tw-linux-x86_64 # [linux]
+    sha256: c0d08bbf3059296e4819d33c6ff9282cfdd4d97087213c2d9aa0b5943ced60f4 # [linux]
+  - url: https://github.com/seqeralabs/tower-cli/releases/download/v{{ version }}/tw-osx-x86_64 # [osx]
+    sha256: c32f9ad48a1a49f1aa62a496b253f24423ae361c1309740f22c429de7e9cc75a # [osx] 
+
+requirements:
+  host:
+    - openjdk >=11,<=20
+  run:
+    - openjdk >=11,<=20
+
+test:
+  commands:
+    - tw --help
+    - tw info
```

Comment:
Not sure how these tests are working, do they expect a 0 exit code? The `tw info` command returns a non-zero exit code if some test has fail.